File: A_I_Wanna_Be_the_Guy.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    cin >> n;

    vector<bool> can(n + 1, false);

    int p;
    cin >> p;
    for (int i = 0; i < p; i++)
    {
        int x;
        cin >> x;
        can[x] = true;
    }

    int q;
    cin >> q;
    for (int i = 0; i < q; i++)
    {
        int x;
        cin >> x;
        can[x] = true;
    }

    for (int i = 1; i <= n; i++)
    {
        if (!can[i])
        {
            cout << "Oh, my keyboard!\n";
            return 0;
        }
    }

    cout << "I become the guy.\n";
    return 0;
}

File: A_I_Wanna_Be_the_Guy.py
t = int(input())
listy=[]
is_done =True
x=list(map(int,input().split()))
y=list(map(int,input().split()))
x.sort()
y.sort()
for i in range(t):
    listy.append([i+1,False])
listy.sort()
for xi in x:
    for e in listy:
        if e[0]==xi and e[1]==False:
            e[1]=True
for yi in y:
    for e in listy:
        if e[0]==yi and e[1]==False:
            e[1]=True

for e in listy:
    if e[1]==False:
        is_done = False
        break
if is_done:
    print("I become the guy.")
else:
    print("Oh, my keyboard!")